src/test/java/rocks/inspectit/gepard/agent/resolver/ConfigurationResolverTest.java

```diff
@@ -70,4 +100,11 @@ private InspectitConfiguration createConfiguration(boolean enabled) {
         new InstrumentationConfiguration(List.of(scope));
     return new InspectitConfiguration(instrumentationConfiguration);
   }
+
+  private InspectitConfiguration createConfiguration(boolean enabled, String methodName) {
+    Scope scope = new Scope(TEST_TYPE.getName(), methodName, enabled);
+    InstrumentationConfiguration instrumentationConfiguration =
+        new InstrumentationConfiguration(List.of(scope));
+    return new InspectitConfiguration(instrumentationConfiguration);
+  }
```

Comment:
You can modify the previous createConfiguration() method, so we don't use almost two identical ones.

```
  private InspectitConfiguration createConfiguration(boolean enabled) {
    return createConfiguration(enabled, null);
  }
```